libs/types-common/src/Data/Id.hs

```diff
@@ -418,6 +419,9 @@ newtype RequestId = RequestId
       ToBytes
     )
 
+defRequestId :: RequestId
```

Comment:
You may use this here as well: https://github.com/wireapp/wire-server/blob/develop/services/proxy/src/Proxy/Env.hs#L65

Reply:
I'm wondering if `N/A` wouldn't be better communicated with a `Maybe RequestId`. :thinking: 

However, this is beyond the scope of this PR.